src/_content/schedule/talks/2024-09-24-13-35-t1-lessons-from-e2e-testing-django-web-applications.md

```diff
@@ -0,0 +1,19 @@
+---
+category: talks
+datetime: 2024-09-24 13:35:00-04:00
+end_datetime: 2024-09-24 14:20:00-04:00
+permalink: /talks/lessons-from-e2e-testing-django-web-applications/
+presenter_slugs:
+- avindra-fernando
+room: Grand Ballroom III
+tags:
+- testing
+title: Lessons from E2E Testing Django Web Applications
```

Comment:
Avindra asked that we change the title to "Lessons from E2E Testing Web Applications". @drewbrew can you make that change here?

Reply:
@emullaney using the suggestion feature might be the easiest way to change it. 

```suggestion
title: Lessons from E2E Testing Web Applications
```

(holy moly this PR is so big it's hard to find/comment on things)